FlashDevelop/Docking/TabbedDocument.cs

```diff
@@ -40,24 +38,14 @@ public CustomFloatWindow(DockPanel dockPanel, DockPane pane, Rectangle bounds)
         private void InitializeComponents()
         {
             FormBorderStyle = FormBorderStyle.Sizable;
-            Icon = new Icon(ResourceHelper.GetStream("FlashDevelopIcon.ico"));
+            Icon = Globals.MainForm.Icon;
             ShowInTaskbar = true;
             Owner = null;
```

Comment:
I don't remember every downside of Owner, but there were several, like the child form is always above the main form, and IIRC when minimizing the main form the child form was also minimized and there were problems related to focus.